src/main/scala-2.10/com/mpc/scalats/core/ScalaParser.scala

```diff
@@ -179,11 +192,11 @@ final class ScalaParser(logger: Logger) {
         IntRef
       case "Long" =>
         LongRef
-      case "Double" =>
+      case "Double" | "BigDecimal" =>
         DoubleRef
       case "Boolean" =>
         BooleanRef
-      case "String" =>
+      case "String" | "UUID" =>
```

Comment:
Same

Reply:
Same as for `DoubleRef`/`BigDecimalRef`... maybe I'll introduce `UuidRef` on the side of ScalaModel, compiling to TS `string`?